### PR TITLE
Set up Github Actions

### PR DESCRIPTION
**Pull request by odisseus**

### CODE REVIEW

File: ci/test.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -x
 
 export IDEPROBE_DISPLAY=xvfb
-export PANTS_PLUGIN_PATH="/tmp/pants.zip"
+#export PANTS_PLUGIN_PATH="/tmp/pants.zip"
```
Comment (lukaszwawrzyk) [inline ci/test.sh:5]: You can actually delete this, and all the logic related to this ENV, as it will be done in a different way

Comment (odisseus) [reply ci/test.sh]: Ok

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+      fail-fast: false
```
Comment (tpasternak) [inline .github/workflows/main.yml:11]: Maybe we should consider enabling this?

Comment (lukaszwawrzyk) [reply .github/workflows/main.yml:11]: I guess it will be better to let it continue in case something else will fail it will help more with fixing.

**Review by lukaszwawrzyk — APPROVED** [review]

**Comment by lukaszwawrzyk** [comment]
It is good to merge. 
It already works better as it took 47 minutes to run everything. Although suites don't run parallel yet. We can address this after we develop building pants from sources). 
It seems like steps cannot run in parallel, only jobs can. We will likely need:
- preparation job to build common docker image that will be reused for tests and upload it with upload-artifact
- test jobs that will run in parallel and depend on job above, will just download the image and start tests

**Pull request merged**